api/analysis/gosec.go

```diff
@@ -94,7 +94,7 @@ func GosecStartAnalysis(CID string, cOutput string) {
 	}
 
 	// step 3: update analysis' cResult into AnalyisCollection.
-	issueMessage := "No issues found."
+	issueMessage := "Warning found."
 	if cResult != "passed" {
```

Comment:
I think `cResult` will never hold the string `passed`, this way this `if` condition will never be used. Could you please confirm that @rafaveira3 ?

Reply:
Good observation. If `cOutput` comes empty, no issues were found from Gosec. We will set that [here](https://github.com/globocom/huskyCI/pull/223/files/ced68a1dbd59f274c24e5e7ebe2cbe91c7285c7d#diff-de4cd9ec349b16ef4f6283ce814f34cdR52) and a `return` call we be made. If we ever manage to get until this code here, we can guarantee that warnings were found.

Reply:
Yes! Good catch, @Krlier ! I have updated this at f22953a , would you mind taking a look at it, please?